core/backend/src/test/imodel/IModel.test.ts

```diff
@@ -1685,6 +1685,20 @@ describe("iModel", () => {
   });
 
   it("should be able to reproject with iModel coordinates to or from any other GeographicCRS", async () => {
+    const addGcsWs = async (id: string) => {
+      try {
+        const ws = await IModelHost.appWorkspace.getContainer({ id });
+        const fileName = ws.localFile;
+        IModelHost.appWorkspace.dropContainer(ws);
+        expect(IModelHost.platform.addGcsWorkspace(fileName)).to.be.true;
+      } catch (e) {
+        // eslint-disable-next-line no-console
+        console.log(`cannot load GCS Workspace: ${id}`);
```

Comment:
If this happens does that mean the test should fail?

Reply:
I get:
```
cannot load GCS Workspace: usa
cannot load GCS Workspace: uk
```

Reply:
no. i made it so the backend *can* find gcs data in a workspace, and i used this to test that. @AlainRobertAtBentley is working on making the workspace files for all the gcs data. But for now we still deliver it all as loose files in the assets directory so these tests pass even though they don't use workspaces.

Ultimately, we'll stop delivering gcs data as "assets". Then, these tests will have to be moved to "integration" tests so they can download the gcs workspaces and test that everything works without the data coming from builds.